### PR TITLE
[change] Edit permissions of /opt/openwisp2 #243

### LINKED ISSUE #243: [change] Edit permissions of /opt/openwisp2 and settings.py
Make `/opt/openwisp2` and any directory created under it not readable, nor executable except for its owner and its group.
Ensure `/opt/openwisp2/openwisp2/settings.py` is only readable by owner and group.

### PR DESCRIPTION
**Pull request by Purhan**
Closes #243

### CODE REVIEW

**Comment by Purhan** [comment]
Permissions for `/opt/openwisp2/openwisp2/settings.py`
image

File: tasks/django.yml [hunk 1]
```diff
@@ -156,3 +156,12 @@
     or "changed" in load_initial_data_result.stdout
   args:
     chdir: "{{ openwisp2_path }}"
+
```
Comment (pandafy) [inline tasks/django.yml:159]: @Purhan you should be able to change permissions in the following code. Hence, removing the need for adding a dedicated task for it. 

https://github.com/openwisp/ansible-openwisp2/blob/766a49bdcd06775a9d333dbfea8d3019d58076d1/tasks/django.yml#L3-L8

I believe the idempotence test is also failing due to this. While running above task during idempotence test, molecule observes a change and hence the test suite fails.

**Review by pandafy — CHANGES_REQUESTED** [review]

File: tasks/django.yml (hunk 1 above)
Comment (Purhan) [reply tasks/django.yml:159]: Thanks! I will quickly implement these changes. One question: the issue doesn't mention anything about write permissions, should it be granted to user?

Comment (nemesisdesign) [reply tasks/django.yml:159]: I have done the changes I needed locally and polished them. Will send them asap. I suggest to pick another issue, thanks!

**Comment by Purhan** [comment]
Closing after discussion on IM.

**Pull request closed without merging**